packages/react-mutation-mapper/src/util/LollipopPlotUtils.ts

```diff
@@ -89,12 +89,18 @@ export function calcYMaxInput(
 
     if (input === undefined) {
         input = yAxisSameScale
-            ? getCommonYAxisMaxSliderValue(
-                  yMaxStep,
-                  countRange,
-                  oppositeCountRange
+            ? parseFloat(
+                  getCommonYAxisMaxSliderValue(
+                      yMaxStep,
+                      countRange,
+                      oppositeCountRange
+                  ).toFixed(Math.log10(yMaxStep) * -1)
```

Comment:
seems like this would pass a negative number to toFixed(), which seems to be against the rule no?